framework/source/class/qx/ui/form/RadioButtonGroup.js

```diff
@@ -287,12 +289,55 @@ qx.Class.define("qx.ui.form.RadioButtonGroup",
      */
     getSelectables: function(all) {
       return this.__radioGroup.getSelectables(all);
+    },
+
+
+    /**
+     * Select given value.
+     *
+     * @param item {null|var} Item to set as selected value.
+     * @return {null|Error} The status of this operation.
+     */
+    setValue : function(item) {
+      if (item && 'object' === typeof item && item instanceof qx.ui.form.IRadioItem) {
+        return this.__radioGroup.setValue(item);
```

Comment:
Hmmm... RadioGroup does not have a `value` property prior to this PR, that I can find, and this PR doesn't add one. How does this work? I think it should say, `this.__radioGroup.setSelection( [ item ] );` shouldn't it? (And similarly elsewhere in here.)

Additionally, my recent investigation shows that in the item itself, it should be possible to use properties other than the item's `value` property as the one that the RadioGroup apples to. This needn't, I believe, affect your PR greatly, as long as it doesn't *depend* on the item having a `value` property. See #9392 for what I'm talking about.

Reply:
Hi @derrell,

This PR includes the ```IField``` interface [here](https://github.com/qooxdoo/qooxdoo/pull/9367/files#diff-a5b5b141c7f0ed3db090c3995b0e9cb0R39). The implementation of this interface was added [here](https://github.com/qooxdoo/qooxdoo/pull/9367/files#diff-7084803c1921aa284f73e74ee39f1c21R68)(1) (```MSingleSelectionHandling``` mixin). ```RadioGroup``` already included this mixin ([see master](https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/ui/form/RadioGroup.js#L43)). That's why it doesn't show up in the diff. While, indeed, the ```RadioGroup``` is not implementing the ```IField``` methods + event directly. It does so by including the ```MSingleSelectionHandling```.

 As for #9392, as seen in the ```MSingleSelectionHandling```, the implementation of the interface relies on the ```SingleSelectionManager``` (see: 1). Therefore, it should not affect this PR.

I hope this clears it up a bit.